WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift

```diff
@@ -188,6 +193,12 @@ private extension BillingInformationViewController {
 
         ServiceLocator.analytics.track(.orderDetailCustomerEmailMenuTapped)
     }
+
+    private func copyEmailHandler() {
+        guard let email = order.billingAddress?.email else { return }
```

Comment:
We usually break the return out on a line by itself

Reply:
Thanks, good spot. I'd misremembered the exception for `guard self = self` as applying to all single line guard statements.